src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,161 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class CalculatorTest {
+    private final double EPS = 1e-9;
+
+    @Test
+    public void simplePositiveAddedTest() {
+        Calculator calculator = new Calculator(2, 3, '+');
+        Assert.assertEquals(5, calculator.calculate(), EPS);
+    }
+
+    @Test
+    public void simpleNegativeAddedTest() {
```

Comment:
```suggestion
    public void negativeAdditionTest() {
```

I like your method names
but let's remove `simple` prefix from them - it doesn't tell us anything

also, please create a test for wrong operator